business_requirement_activity_report/report/timesheet_activity_report.py

```diff
@@ -1,126 +1,144 @@
-                        -- Link with the task
-                        LEFT OUTER JOIN project_task_work tw
-                            ON tw.hr_analytic_timesheet_id = ts.id
-                        LEFT OUTER JOIN project_task t ON t.id = tw.task_id
-                        -- Link with the issue
-                        LEFT OUTER JOIN project_issue i ON i.id = ts.issue_id
-                        -- Link with the project
-                        LEFT OUTER JOIN project_project p
-                            ON p.id = COALESCE(t.project_id, i.project_id)
-                        -- Link with the analytic account
-                        LEFT OUTER JOIN account_analytic_account a
-                            ON a.id = p.analytic_account_id
-                        -- Link with the BR
-                        LEFT OUTER JOIN business_requirement b
-                            ON b.linked_project = p.id
-                ) AS q
-            )""")
+# -*- coding: utf-8 -*-
+#   2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import fields, models
+from openerp import tools
+
+
+class TimesheetReport(models.Model):
+    """ Timesheet Activities Report """
+    _name = "timesheet.activity.report"
+    _auto = False
+    _description = "Timesheet Activities Report"
+    _rec_name = 'activity_name'
+
+    id = fields.Integer(string='ID', readonly=True)
+    activity_type = fields.Selection(
+        selection=[
+            ('task', 'Task'),
+            ('issue', 'Issue'),
+            ('timesheet', 'Timesheet'),
+        ],
+        string='Type',
+        readonly=True,
+        help="Type is used to separate Tasks, " +
+        "Issues and Timesheets input directly")
+    description = fields.Char(string='Description')
+    hours = fields.Float(string='Time spent')
+    user_id = fields.Many2one(
+        comodel_name='res.users',
+        string='User',
+        ondelete='restrict',
+        readonly=True)
+    product_id = fields.Many2one(
+        comodel_name='product.product',
+        string='Product',
+        ondelete='restrict',
+        readonly=True)
+    date = fields.Date('Date', readonly=True)
+    project_id = fields.Many2one(
+        comodel_name='project.project', string='Project', readonly=True)
+    project_state = fields.Char(
+        string='State', readonly=True, help="Project State")
+    activity_stage_id = fields.Many2one(
+        comodel_name='project.task.type',
+        string='Stage',
+        readonly=True,
+        help="Activity Stage")
+    account_id = fields.Many2one(
+        comodel_name='account.analytic.account',
+        string='Analytic account',
+        readonly=True)
+    activity_id = fields.Char(
+        string='Activity id', readonly=True, help="Task id or Issue id")
+    activity_name = fields.Char(
+        string='Activity name', readonly=True, help="Task name or Issue name"),
+    task_categ_id = fields.Many2one(
```

Comment:
Yes I would use asap this field related to a new module we recently proposed to https://github.com/OCA/project/pull/225 or just drop it to merge this PR the soon the better.